site/utils/connectors.js

```diff
@@ -5,7 +5,7 @@ import { WalletLinkConnector } from '@web3-react/walletlink-connector'
 const environment = process.env.NEXT_PUBLIC_ENVIRONMENT
 const nodeUrl = process.env.NEXT_PUBLIC_NODE_URL
 export const injected = new InjectedConnector({
-  supportedChainIds: [1, 137, 1337, 31337, 43114]
+  supportedChainIds: [1, 137, 1337, 31337, 43114, 743111]
```

Comment:
@gabmontes for the Hemilabs PF versions, should we support anything besides Hemi Testnet (and eventually, hemi Mainnet)? 

I wonder if for this repository, this should be like

```suggestion
  supportedChainIds: [743111/*, HemiMainnetId */]
```

Reply:
Made this configurable via envs here: https://github.com/hemilabs/pure.finance/pull/2/commits/1c0a792be1d0b294cee93ac808edb62ac1b226a9

Reply:
still in doubt if `1, 137, 1337, 31337, 43114` should be removed

Reply:
you're right, I wrongly commented here, this has nothing to do with the envs added in https://github.com/hemilabs/pure.finance/commit/1c0a792be1d0b294cee93ac808edb62ac1b226a9 😅, I'll leave it as `[743111]` for now

Reply:
We should only support Hemi and Hemi testnet in this fork.

Reply:
Changed here: https://github.com/hemilabs/pure.finance/pull/2/commits/50613c94ab27cb3cc111f820f6ba587e14cf0d97